modules/develop/pages/integrations.adoc

```diff
@@ -0,0 +1,343 @@
+= Integrations, Connectors, and Tools
+:page-toclevels: 2
+:page-topic-type: reference
+:description: Integrate Couchbase with your development ecosystem and production and deployment tools.
+
+
+[abstract]
+{description}
+
+Third-party and community products and integrations -- 
+although not by and large supported by Couchbase (see <<#support-model,Support Model section>>) --
+are a vital tool for most larger development projects.
+
+These are examples of integrations that Couchbase or partners have developed to enhance Couchbase, Capella, and partner services.
+
+
+[#partner-integrations-with-couchbase]
+== Partner Integrations
+
+
+.Partner Integrations List
+[cols="20,45,50,20,20,28"]
+|===
+| Integration | Summary | Links | Capella | Self-managed |  <<#support-model,Support Model>>
+
+
+| Netlify
+| Use Couchbase Capella and Netlify Serverless Functions to handle dynamic content without needing infrastructure.
+a| 
+* https://www.netlify.com/integrations/couchbase/[Netlify partner page]
+* https://developer.couchbase.com/tutorial-quickstart-netlify[Tutorial]
+| ✔
+| ✖
+| Couchbase Officially Supported
+
+
+| Vercel
+| Easy build and deploy for your applications.
+a|
+* https://vercel.com/integrations/couchbase-capella[Vercel partner page & tutorial]
+* https://www.couchbase.com/blog/integrating-vercel-with-couchbase-capella[blog]
+| ✔
+| ✖
+| Couchbase Officially Supported
+
+
+| Molo17 GlueSync
+| Real-time, cloud-native data replication suite for NoSQL.
+a| 
+* https://www.couchbase.com/partners/molo17/[Intro & Tutorial]
+* https://molo17.com/gluesync/[GlueSync homepage]
+| ✔
+| ✔
+| Partner Supported
+
+
+| ODBC / JDBC
+| Easily access live Couchbase data from BI, Analytics, Reporting, ETL, & Custom Apps.
+a|
+* https://docs.couchbase.com/server/current/connectors/odbc-jdbc-drivers.html[Couchbase ODBC and JDBC Drivers]
```

Comment:
Should this be an xref or link? It's in the docs?

Reply:
:+1: